hicexplorer/hicMergeMatrixBins.py

```diff
@@ -154,7 +154,7 @@ def merge_bins(hic, num_bins):
     Parameters
     ----------
 
-    hic : HiCMatrix object
+    hic : Hi-C Matrix object
```

Comment:
Here we don't need to replace by Hi-C Matrix because the name of the class is HiCMatrix.